controllers/error-handling-controllers.js

```diff
@@ -1,4 +1,22 @@
+exports.handle400Errors = (err, req, res, next) => {
+  if (err === 'Invalid article ID') {
+    console.log(400, '============');
+    console.log(err);
+    res.status(400).send({ msg: 'Invalid article ID' });
+  } else next(err);
+};
+
+exports.handle404Errors = (err, req, res, next) => {
+  if (err === 'Article not found') {
+    console.log(404, '============');
+
+    console.log(err);
+    res.status(404).send({ msg: 'Article not found' });
+  } else next(err);
+};
+
 exports.handle500Errors = (err, req, res, next) => {
+  console.log(500, '============');
```

Comment:
This is the only place we should really leave a console.log (for debugging) please remove any others before merging.